docs/suspensive.org/src/pages/docs/react-query/tanstack-query-compatibility.en.mdx

```diff
@@ -68,47 +72,55 @@ npx suspensive-react-query --help
 
 ```bash
 npx suspensive-react-query status
 ```
 
-For instance, if using @suspensive/react-query@2.4.0 and @tanstack/react-query@5.49.2, you might see,
+For instance, if using @suspensive/react-query@2.9.0 and @tanstack/react-query@5, the result might look like this:
 
 ```
-Suspensive React Query status:
-@suspensive/react-query@2.4.0 exports @suspensive/react-query-5's interfaces
-
-- SuspenseQuery
-- SuspenseQueries
-- SuspenseInfiniteQuery
-- QueryErrorBoundary
-
-@tanstack/react-query@5.49.2
-
-The versions are compatible.
+┌─────────────────────────┬─────────────────────────────────┬────────┬────────────────────────────────────────────────────────┐
+│ @suspensive/react-query │ result                          │ status │ advice                                                 │
+├─────────────────────────┼─────────────────────────────────┼────────┼────────────────────────────────────────────────────────┤
+│ version                 │ 2.9.0                           │ 🟢     │                                                        │
+├─────────────────────────┼─────────────────────────────────┼────────┼────────────────────────────────────────────────────────┤
+│ export                  │ @suspensive/react-query-5@2.9.0 │ 🟢     │                                                        │
+├─────────────────────────┼─────────────────────────────────┼────────┼────────────────────────────────────────────────────────┤
+│ peerDependency          │ @tanstack/react-query@5         │ 🟢     │ The versions are compatible.                           │
+├─────────────────────────┼─────────────────────────────────┼────────┼────────────────────────────────────────────────────────┤
+│ You can use             │ <SuspenseQuery/>                │ 🟢     │ For more detailed information about the provided APIs, │
+│                         │ <SuspenseQueries/>              │        │ please visit the official documentation:               │
+│                         │ <SuspenseInfiniteQuery/>        │        │ https://suspensive.org/docs/react-query/motivation     │
+│                         │ <Mutation/>                     │        │                                                        │
+│                         │ <QueryErrorBoundary/>           │        │                                                        │
+└─────────────────────────┴─────────────────────────────────┴────────┴────────────────────────────────────────────────────────┘
 ```
 
 ### switch
 
-Switches to the @suspensive/react-query interface compatible with @tanstack/react-query. For example, if using @tanstack/react-query v5 and encountering issues, you can switch to a compatible interface with,
+Switches to a different version of the @suspensive/react-query interface based on the @tanstack/react-query version. Use caution as this can force a change that may not be compatible with your current setup.
```

Comment:
```suggestion
Update entry file(dist/index.js) of the @suspensive/react-query. 

```js
// When you use "npx suspensive-react-query switch 4"
// dist/index.js of @suspensive/react-query
export * from '@suspensive/react-query-4' // Suspensive interfaces for @tanstack/react-query@4

// When you use "npx suspensive-react-query switch 5"
// dist/index.js of @suspensive/react-query
export * from '@suspensive/react-query-5' // Suspensive interfaces for @tanstack/react-query@5
```

<Callout type="warning">

Use caution as this can force a change that may not be compatible with your current setup.

</Callout>

```